cscs-checks/prgenv/cuda/cuda_samples.py

```diff
@@ -8,14 +8,18 @@
 import reframe.utility.osext as osext
 
 
-class CudaSamples(rfm.RegressionTest):
-    def __init__(self):
+@rfm.parameterized_test(['deviceQuery'], ['concurrentKernels'],
+                        ['simpleCUBLAS'], ['bandwidthTest'],
+                        ['conjugateGradientCudaGraphs'])
+class CudaSamplesTest(rfm.RegressionTest):
+    def __init__(self, test_name):
+        self.descr = f'CUDA {test_name} test'
```

Comment:
The new parameter syntax injects the parameters directly as class attributes
```suggestion
        self.descr = f'CUDA {self.test_name} test'
```